data/kaushalnitb.json

```diff
@@ -0,0 +1,18 @@
+  {
+  "name": "Kaushal Kishor",
+  "displayStatsPublic": true,
```

Comment:
After a recent update, this code is no longer needed. You can delete this line `"displayStatsPublic": true,`

Reply:
I Have updated the code.Kindly verify it now.